module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -473,33 +517,49 @@ namespace module::behaviour::strategy {
     }
 
     void SoccerStrategy::goalieWalk(const Field& field, const Ball& ball) {
+
+        // make a motionCommand to emit
         std::unique_ptr<MotionCommand> motionCommand;
 
+        // time since the ball has been seen, in microseconds, current clock - time the ball was last measured
         float timeSinceBallSeen =
             std::chrono::duration_cast<std::chrono::microseconds>(NUClear::clock::now() - ballLastMeasured).count()
-            * 1e-6;
+            * 1e6;  // SUSS
+
+        // timeSinceBallSeen < 1 (as stated in the cfg file SoccerStrategy.yaml)
         if (timeSinceBallSeen < cfg_.goalie_command_timeout) {
 
+            // position == Hfw
             Eigen::Affine2d position(field.position);
+
             float fieldBearing  = Eigen::Rotation2Dd(position.rotation()).angle();
             int signBearing     = fieldBearing > 0 ? 1 : -1;
             float rotationSpeed = -signBearing
```

Comment:
I can work out what this passage does, but I have to think about it a bit. Idk. If we could make it slightly easier to follow, that would be good, but :shrimp:
```suggestion
            const float fieldBearing  = Eigen::Rotation2Dd(position.rotation()).angle();
            const int signBearing     = fieldBearing > 0 ? -1 : 1;
            const float rotationSpeed = signBearing
```